readme.md

```diff
@@ -50,6 +57,10 @@ Add the following to your POM file:
          <useTestNG>false</useTestNG>
          <!-- The naming scheme to use for the generated test classes.  One of 'simple' or 'feature-title' --> 
          <namingScheme>simple</namingScheme>
+         <!-- Generate ReRun runners instead of JUnit and TestNG. --> 
+         <useReRun>true</useReRun>
```

Comment:
I didn't try for TestNG. Once i'm done with current changes. Will look for TestNG as well. :)